crates/bevy_core_pipeline/src/tonemapping/tonemapping_shared.wgsl

```diff
@@ -35,3 +221,80 @@ fn screen_space_dither(frag_coord: vec2<f32>) -> vec3<f32> {
     dither = fract(dither.rgb / vec3<f32>(103.0, 71.0, 97.0));
     return (dither - 0.5) / 255.0;
 }
+
+fn tone_mapping(in: vec4<f32>) -> vec4<f32> {
+    var color = max(in.rgb, vec3(0.0));
+
+    //let luma = powsafe(vec3(tonemapping_luminance(color)), 1.0); // highlight gain gamma: 0..
+    //color += color * luma.xxx * 1.0; // highlight gain: 0.. 
+
+    color = saturation(color, view.color_grading.pre_saturation);
+    color = powsafe(color, view.color_grading.gamma);
+    color = color * powsafe(vec3(2.0), view.color_grading.exposure);
+    color = max(color, vec3(0.0));
+
+    // tone_mapping
+#ifdef TONEMAP_METHOD_NONE
+    color = color;
+#endif
+#ifdef TONEMAP_METHOD_REINHARD
+    color = tonemapping_reinhard(color.rgb);
+#endif
+#ifdef TONEMAP_METHOD_REINHARD_LUMINANCE
+    color = tonemapping_reinhard_luminance(color.rgb);
+#endif
+#ifdef TONEMAP_METHOD_ACES
+    // TODO figure out correct value for white here, or factor it out
+    color = tonemapping_aces_godot_4(color.rgb, 1000.0);
+#endif
+#ifdef TONEMAP_METHOD_AGX
+    color = applyAgXLog(color);
+    color = applyLUT3D(color, 32.0, vec2<f32>(1024.0, 32.0), vec2(0.0));
+#endif
+#ifdef TONEMAP_METHOD_SBDT
+    color = tonemapping_sbdt(color.rgb);
+#endif
+#ifdef TONEMAP_METHOD_SBDT2
+    let block_size = 32.0;
+    var c = color.rgb;
+    let LUT_EV_RANGE = vec2(-11.0, 8.0);
+    c = (log2(c) - LUT_EV_RANGE[0]) / (LUT_EV_RANGE[1] - LUT_EV_RANGE[0]);//convertOpenDomainToNormalizedLog2(c, -11.0, 8.0);
```

Comment:
If this is correct? Or why is this comment here?:
```suggestion
    // LUT alternative to: convertOpenDomainToNormalizedLog2(c, -11.0, 8.0);
    c = (log2(c) - LUT_EV_RANGE[0]) / (LUT_EV_RANGE[1] - LUT_EV_RANGE[0]);
```

Reply:
This was was for something old. Not relevant now.